src/main/webapp/vendor/startbootstrap/planner.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
```

Comment:
i think @savsa was looking into sharing the header/base template for a page. Any chance we could figure that out soon? If not, let's file an issue to handle it soon so we don't have to maintain two versions of this.